src/Request/Cart/PickupCartRequest.php

```diff
@@ -31,6 +30,26 @@ public function __construct(Request $request)
      */
     public function getCommand(): CommandInterface
     {
-        return new PickupCart($this->token, $this->channel);
+        return new PickupCart($this->token, $this->channelCode);
+    }
+
+    /**
+     * {@inheritdoc}
+     *
+     * @return CartByToken
+     */
+    public function getQuery(): QueryInterface
```

Comment:
I was against this solution at first but could be really easy to use, so let's test it 👌